backup/backup_remote

```diff
@@ -54,7 +54,7 @@ if ! $(ssh $SERVER "[ -d $BACKUP_DIR ]"); then
 fi
 
 NOW=$(date +'%Y_%d_%m-%H_%M_%S')
-BACKUP_PATH="$BACKUP_DIR/$NOW.tar"
+BACKUP_PATH="$BACKUP_DIR/$HOSTNAME_$NOW.tar"
```

Comment:
I've included the hostname in the resultant file too so that now we can backup to a remote we know the origin of the backup